.github/workflows/chuffed.yml

```diff
@@ -37,9 +36,9 @@ jobs:
             ~/.cargo/git/db/
             target/            
             solvers/chuffed/vendor
-          key: ${{ runner.os }}-chuffed-${{ steps.chuffed-cache.outputs.chuffed_sha }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-          # match chuffed exactly, cargo inexactly
-          restore-keys: ${{ runner.os }}-chuffed-${{ steps.chuffed-cache.outputs.chuffed_sha}}-cargo-
+
+          key: stable-${{ runner.os }}-gitmodules-${{ steps.cache-vars.outputs.submodule_sha }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
maybe more important, should this not say chuffed somewhere in the key? With my suggestion I would try calling this just ubuntu-chuffed. I am not even sure if we need the stable/nightly distinction.

Reply:
i was thinking that both chuffed and minion could share a cache, as we will need both at the same time anyways

Reply:
oh thats a typo, misread it